packages/grant-explorer/src/features/discovery/RoundDaysDetails.tsx

```diff
@@ -0,0 +1,76 @@
+import tw from "tailwind-styled-components";
+
+export function getRoundDaysText({
+  roundStartsIn,
+  roundEndsIn,
+}: {
+  roundStartsIn?: number;
+  roundEndsIn?: number;
+}) {
+  const days = pluralize(["day", "days"]);
+
+  if (roundStartsIn && roundStartsIn > 0) {
+    return `Starts in ${roundStartsIn} ${days(roundStartsIn)}`;
+  }
+  if (roundEndsIn === 0) return "Ends today";
+  if (!roundEndsIn) return "No round end date";
+
+  return roundEndsIn > 0
+    ? `${roundEndsIn} ${days(roundEndsIn)} left in round`
+    : `Ended ${-roundEndsIn} ${days(-roundEndsIn)} ago`;
+}
+
+export function getRoundApplicationDaysText({
+  applicationsStartsIn,
+  applicationsEndsIn,
+}: {
+  applicationsStartsIn?: number;
+  applicationsEndsIn?: number;
+}) {
+  const days = pluralize(["day", "days"]);
```

Comment:
`pluralize()` doesn't actually pluralize its arguments so it's misguiding. Not sure how to call it though — I'd think `createPluralizer`, but what it creates isn't a pluralizer (something that takes a work and produces its plural) either.

Reply:
Renamed to:
```tsx
const dayTerm = getSingularPlural(["day", "days"]);
```